tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -166,12 +166,6 @@ void TangoRosNode::onInit() {
   area_description_T_start_of_service_publisher_ =
       node_handle_.advertise<geometry_msgs::TransformStamped>(
           AREA_DESCRIPTION_T_START_OF_SERVICE_TOPIC_NAME, queue_size, latching);
-  point_cloud_publisher_ =
```

Comment:
(Including @meyerj since he initially reported the issue)

@smits With this PR the parameter setting has influence on the advertisement each time we connect to Tango.

The reason behind this change was not to gain performance but to reduce risks of user mistakes. If a topic is listed in `rostopic list` and the camera is actually disable it can be confusing for the user.